src/Components/ItemList/ItemList.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Item from '../Item/Item';
+
+
+const ItemList = ({ allItems }) => (<ul>
+    	{allItems.map(item => (
+    		<li key={item.value}> 
+    			<Item value = {item.value} /> 
+    		</li> ))}	
```

Comment:
Codacy found an issue: [Mixed spaces and tabs.](https://app.codacy.com/gh/WebHeroSchool/todo_react_project/pullRequest?prid=5893381)